apps/dashboard/assets/webpack.config.js

```diff
@@ -1,29 +1,20 @@
 const path = require('path');
```

Comment:
We can probably delete the dashboard app?

Reply:
Sure, I can do that